spec.html

```diff
@@ -40,12 +40,12 @@ <h1>ApplyOptionsToTag( _tag_, _options_ )</h1>
           1. Set _tag_ to _tag_ with the substring corresponding to the `unicode_language_subtag` production replaced by the string _language_.
         1. If _script_ is not *undefined*, then
           1. If _tag_ does not contain a `unicode_script_subtag` production, then
-            1. Set _tag_ to the concatenation of the `unicode_language_subtag` production of _tag_, `"-"`, _script_, and the rest of _tag_.
+            1. Set _tag_ to the concatenation of the `unicode_language_subtag` production of _tag_, `"-" unicode_script_subtag`, and the rest of _tag_.
```

Comment:
Here `script` is actually referring to the variable `script` and not to the "script" production.